.github/workflows/tests.yml

```diff
@@ -121,6 +121,10 @@ jobs:
         LOG_DIR: k8s-cluster-state
       run: |
         docker run --rm -v $(pwd):/cdk -v $(pwd)/agent_logs:/domino-deployer/logs -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -e KUBECONFIG=/cdk/kubeconfig $DEPLOYER_IMAGE python -m fleetcommand_agent run -f /cdk/domino.yml
+    - name: install-kubectl
+      uses: azure/setup-kubectl@v2.0
```

Comment:
Gerardo pinned kubectl in the past day or so, so you should probably rebase. I don't have an opinion on which install method to go with, but we'll be doing it twice if this merges.